balancer-labs/frontend-v2
# Hofix: Remove synthetix asset warning

# Description

Removes the warning seen on pools such as https://app.balancer.fi/#/pool/0x072f14b85add63488ddad88f855fda4a99d6ac9b000200000000000000000027 about this bug: https://forum.balancer.fi/t/medium-severity-bug-found/3161 as the bug has now been fixed. 

## Type of change


## How should this be tested?

- Ensure that the banner doesn't appear on any of the pools in the list and that they load correctly. 

## Checklist:
